apps/dashboard/src/components/primitives/form/form.tsx

```diff
@@ -120,8 +120,8 @@ const FormMessagePure = React.forwardRef<
       className={formMessageVariants({ variant: error ? 'error' : 'default', className })}
       {...props}
     >
-      {error ? <RiErrorWarningFill className="size-4" /> : <RiInformationFill className="size-4" />}
-      <span className="mt-[1px] text-xs leading-3">{body}</span>
+      <span>{error ? <RiErrorWarningFill className="size-4" /> : <RiInformationFill className="size-4" />}</span>
+      <span className="mt-[1px] text-xs leading-4">{body}</span>
```

Comment:
The icon is still a bit suspicious - it would be ok if the content would be shorter.

Before:

![image](https://github.com/user-attachments/assets/5c53dfa9-f453-46f6-8c96-c0b915c4aa72)

After:

![image](https://github.com/user-attachments/assets/5e46fa9a-7dc6-45f6-bea1-3d5715afbe4e)